xml/deployment_installation.xml

```diff
@@ -899,10 +897,11 @@ network-interfaces: |
 #cloud-config
 debug: True
 disable_root: False
+ssh_deletekeys: False
 ssh_pwauth: True
 chpasswd:
   list: |
-    root:<replaceable>root</replaceable>
+    root:<replaceable>MyPassword</replaceable>
```

Comment:
MY_PASSWORD (replacables are written in caps)

Reply:
Done.

Reply:
The team hasn't agreed on a solution yet.

Reply:
OK, fair enough. Are you happy with my amendments to this at present, though, or do you want more changes?